### PR TITLE
Add Transparency Report to blog

### PR DESCRIPTION
**Pull request by olasitarska**
Also fixing readibility of long texts on our blog

### CODE REVIEW

File: tinyblog/articles/2016-04-11-coc-transparency-report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+We believe that transparency on this subject is important in order to give everyone some visibility on what happens “behind the scenes”. We want to show everyone why our CoC is important but also how it is enforced in practice.
+
+We are also sharing reports for all of these incidents with the [DSF Code of Conduct committee](https://www.djangoproject.com/foundation/committees/).
+
+We hope that by making this list public, it will encourage more people to report similar incidents and in turn make our conference a better, more inclusive space for everyone.
+
+We look forward to seeing other events publishing their own CoC reports and we welcome any feedback or ideas on how to improve our CoC handling next year.
+
+Much love,  
+DjangoCon Europe CoC team
```
Comment (bmispelon) [inline tinyblog/articles/2016-04-11-coc-transparency-report.md:38]: Aren't we officially the "CoC Active Response Effort" now? :)

Comment (olasitarska) [inline tinyblog/articles/2016-04-11-coc-transparency-report.md:38]: I find that quite a mouthful, and not really explaining a lot :)

File: tinyblog/articles/2016-04-11-coc-transparency-report.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- During the conference speakers dinner, two of attendees made an inappropriate joke towards another attendee. The two attendees were reminded that not all jokes are appropriate during official conference events.
+- Person identified as not attending the conference posted an inappropriate tweet mentioning the conference official account. The person was politely asked to remove that tweet, which resulted in a series of offending messages and physical threats sent to a member of Code of Conduct team by that person. The Code of Conduct team decided to raise the issue with the DSF Code of Conduct committee, asking them to ban this person from attending future Django events and send them an email informing of actions taken.
+- One of the lightning talks contained unfortunate phrasing directed towards one of the attendees. Video from that lightning talk won’t be published online.
+We are not publishing this list to shame anyone or pat ourselves on the back.
```
Comment (oinopion) [inline tinyblog/articles/2016-04-11-coc-transparency-report.md:27]: Add new line before "We are not", otherwise it just merges with last bullet point

Comment (olasitarska) [inline tinyblog/articles/2016-04-11-coc-transparency-report.md:27]: Good catch, pushed

**Comment by oinopion** [comment]
Good stuff! :shipit:

**Pull request merged**